### PR TITLE
Adding mvlearn

### PR DESCRIPTION
**Pull request by gmischl1**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

I am ok with being listed as a maintainer, and rflperry has commented below.

### CODE REVIEW

**Comment by rflperry** [comment]
I'd like to be added as a maintainer.

File: recipes/mvlearn/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```
Comment (synapticarbors) [inline recipes/mvlearn/LICENSE:1]: I do not believe you need to manually package the license file since it's included in the tarball from github

Comment (gmischl1) [reply recipes/mvlearn/LICENSE]: Removed the LICENSE file, thanks!

**Comment by gmischl1** [comment]
@synapticarbors, ready for review again!

**Pull request merged**